custom_components/ble_monitor/sensor.py

```diff
@@ -1027,10 +1026,11 @@ async def async_added_to_hass(self):
             self.ready_for_update = True
             return
         old_state = await self.async_get_last_state()
+        _LOGGER.info(old_state)
         if not old_state:
             self.ready_for_update = True
             return
-        self._state = old_state.state
+        self._state = True if old_state.state == "on" else False
```

Comment:
I propose to change this to `self._state = True if old_state.state == STATE_ON else False`

Reply:
Yes, of course! I keep forgetting about these constants...